src/org/opensolaris/opengrok/analysis/Ctags.java

```diff
@@ -237,6 +238,24 @@ private void initialize() throws IOException {
             command.add("--regex-pascal=/^(uses|interface|implementation)$/\\1/s,Section/");
             command.add("--regex-pascal=/^unit[[:space:]]+([a-zA-Z0-9_<>, ]+)[;(]/\\1/u,unit/");
             
+            // PowerShell
+            command.add("--langdef=Posh");
+            command.add("--langmap=Posh:+.ps1,Posh:+.psm1");
+            command.add("--kinddef-Posh=f,function,functions");
```

Comment:
so we will restrict kinddef only when universal ctags are detected (see other ifs around this area)
for exuberant ctags the semantic info will be simply missing, alt. you might try to rewrite (if possible) to old approach (which is just a loger line from the quick look I had at http://docs.ctags.io/en/latest/news.html?highlight=kinddef#defining-a-kind )

Reply:
and I am also wondering if this restriction change will get the travis build working ... (even though if any tests will depend on this info they might fail, but then I didn't see any tests added ... yet ;) )

Reply:
it actually looks like this is the problem and not the NPEs from auth plugins ... 
so I think once you surround this block of code with
 if (!env.isUniversalCtags()) {
travis will start working ...